src/sentry/workflow_engine/endpoints/validators/base/detector.py

```diff
@@ -33,6 +34,7 @@ class BaseDetectorTypeValidator(CamelSnakeSerializer):
     )
     type = serializers.CharField()
     config = serializers.JSONField(default={})
+    owner = ActorField(required=False, allow_null=True)
```

Comment:
Should it be possible to create a detector without an owner? Right now in the UI we assume that detectors without an owner are created by Sentry

Reply:
🤷‍♂️